spec/support/test_ledger/operation/full_update.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require_relative '../operation'
+
+module LedgerSync
+  module Ledgers
+    module TestLedger
+      class Operation
+        class FullUpdate
```

Comment:
Also I would say `FullUpdate` is very QBO specific. I would drop this completely and just leave `Update`